### PR TITLE
add section on or_other

### LINKED ISSUE #617: Add `or_other` to select widgets section
http://xlsform.org/#specify-other

### PR DESCRIPTION
**Pull request by adammichaelwood**
closes #617 

#### What is included in this PR?

Small section on including "or_other" as an option.

### CODE REVIEW

File: odk1-src/form-question-types.rst [hunk 1]
```diff
@@ -1043,8 +1043,20 @@ To include images as choices for select questions:
      make sure the media folder is placed in
      :file:`/sdcard/odk/forms/`.
 
-  
+.. _or-other:  
+
+Including "other" as an option
```
Comment (yanokwa) [inline odk1-src/form-question-types.rst]: s/option/choice.

It's not clear when you use option vs choice, but we should probably default to choice.

File: odk1-src/form-question-types.rst [hunk 2]
```diff
@@ -1043,8 +1043,20 @@ To include images as choices for select questions:
...
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+On the **survey** sheet, in the :th:`type` column,
+after the type and the list_name,
+you can add :tc:`or_other`.
+This will add "Other" as an additional option to your choice list.
+The :th:`name` value of the choice when selected will be :tc:`other`.
 
+.. seealso:: `Specify other <http://xlsform.org/#specify-other>`_
```
Comment (yanokwa) [inline odk1-src/form-question-types.rst]: The content at this link doesn't add more, so I'd probably skip this.

File: odk1-src/form-question-types.rst [hunk 3]
```diff
@@ -1043,8 +1043,18 @@ To include images as choices for select questions:
...
-  
+.. _or-other:  
+
+Including "other" as a choice
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
+On the **survey** sheet, in the :th:`type` column,
+after the type and the list_name,
+you can add :tc:`or_other`.
+This will add "Other" as an additional option to your choice list.
```
Comment (yanokwa) [inline odk1-src/form-question-types.rst]: Did you intentionally leave option here? It's not bad (maybe a wee bit 😂), I'm just curious.

Comment (adammichaelwood) [reply odk1-src/form-question-types.rst]: I did! (I almost made a note about it, too.)

**Pull request merged**